test/trento/users_test.exs

```diff
@@ -0,0 +1,199 @@
+defmodule Trento.UsersTest do
+  use Trento.DataCase
+
+  alias Trento.Users
+  alias Trento.Users.User
+  import Trento.Factory
+
+  describe "users" do
+    test "list_users returns all users except the deleted ones" do
+      %{id: user_id} = insert(:user)
+      create_deleted_user()
```

Comment:
I don't think we need a function for this. Maybe better to put `insert(:user, deleted_at: DateTime.utc_now())` in the code itself. I find it more declarative

Reply:
done!